collect_app/src/main/java/org/odk/collect/android/utilities/PlayServicesUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.odk.collect.android.utilities;
+
+import android.app.Activity;
+import android.content.Context;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+
+/**
+ * Created by Divya on 3/2/2017.
+ */
+
+public class PlayServicesUtil {
+
+    public static int PLAY_SERVICE_ERROR_REQUEST_CODE = 1001;
+
+    private static GoogleApiAvailability googleApiAvailability;
+    private static int resultCode;
+
+    public static boolean checkPlayServices(Context context) {
+
+        googleApiAvailability = GoogleApiAvailability.getInstance();
+
+        resultCode = googleApiAvailability.isGooglePlayServicesAvailable(context);
+
+        if (resultCode != ConnectionResult.SUCCESS) {
```

Comment:
You can just `return resultCode == ConnectionResult.SUCCESS` here! If it's true, true is returned. If it's false, false is returned. "Boolean zen" 😊